modules/enabling-scanner-v4-operator.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-operator_{context}"]
+= Enabling RHACS Scanner V4 when installing with the Operator
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable it during installation.
```

Comment:
Can it also be enabled after installation?

Reply:
Well, yes, but that is a different section - this is the "during installation" section? We need an abstract for docs.redhat.com that states what the purpose of the section is, so it basically repeats the title.